Sources/XS2AiOS/Protocols/WizardStep.swift

```diff
@@ -1,6 +1,17 @@
 public enum WizardStep: String {
 	case tan
 	case bank
-	case account
+	case accounts
```

Comment:
Hi @fritzfr, I was testing this change and found out that the received json has:
```
  "step" : "accounts",
  "callback" : "account"
```
It looks like we need both `account` and `accounts` cases. Or the `step` and and `callback` properties needs to be mapped to different enums.

Reply:
You are right. Just reverted this to the old version.